designs/UserWideConfiguration-AdditionalExtensibilityFolders.md

```diff
@@ -0,0 +1,78 @@
+
+# Additional user wide configuration options
+
+* Status: In Review
+* Author(s): [Nikolche Kolev](https://github.com/nkolev92)
+* Issue: [9394](https://github.com/NuGet/Home/issues/9394) NuGet should support multiple config files in %APPDATA%\NuGet directory
+
+## Problem Background
+
+The XAML and WinForms teams want to allow control library installers to add user-specific NuGet fallback folders by having their SDK installers drop config files in %APPDATA%\NuGet. On uninstall this file would be deleted.
+
+We considered two alternatives which I mentioned as workarounds above:
+
+1. Require SDK installers to drop config files in %ProgramFiles(x86)%\NuGet\Config instead. However, we've gotten feedback from control vendors that they would like the option of installing without elevation which prevents the installer from being able to write to this directory.
+1. Require SDK installers to modify the one %APPDATA%\NuGet\NuGet.config file. However this seems error prone.

```

Comment:
Another possible destination would be `%ProgramData%` folder (typically, `C:\ProgramData`). Non-admin users can write to it, and all users can read from it.

Reply:
What I do not know what would be Linux/OSX equivalent? Something under `/usr`?

Reply:
That's certainly an option. Familiarity was the key motivator for proposing AppData. 

The programdata equivalent on linux is `/etc/opt`

Reply:
/etc/opt on my Ubuntu on WSL is not user writable.